<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="download_csv">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Загрузить в csv файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить данные Хаба на сервер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_hab_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить данные с Хаба</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data_to_server">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить свои данные на сервер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_data_from_server">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>660</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить данные с сервера</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
